src/gtasa/effects/custom/player/AutoCollectPickupsEffect.cpp

```diff
@@ -0,0 +1,111 @@
+#include "util/EffectBase.h"
+#include "util/GameUtil.h"
+
+#include <CPickups.h>
+
+namespace

```

Comment:
This namespace can be put in the class as well, above the `public:` part as these classes are only instantiated once
(That's also why it's important to reset values like `wait` or similar inside the `OnStart` method)

Reply:
It is not neccessery in my opinion.
The anonymous (unnamed) namespace will let the compiler know that the functions and variables there in will only be used in current instance (a cpp file).
A small amount of compile optimization.

Reply:
Oh interesting, I didn't know that

In that case we can keep it that way :)